docs/components/tree-list.md

```diff
@@ -147,6 +147,30 @@ const items = ref<TreeListItem[]>([
 </script>
 ```
 
+### group
+
+To drag elements from one list into another, both lists must have the same `group` value. Defaults to `k-tree-list` (meaning that elements from one list on a page can be dragged into another, unless a different `name` value is provided for one of them).
+
+In the example below, try toggling grouping on and off and dragging items from one list into another.
+
+<KComponent
+  v-slot="{ data }"
+  :data="{ grouping: true }"
+>
+  <KInputSwitch
+    v-model="data.grouping"
+    label="Grouping"
```

Comment:
"Grouping" is a bit confusing, can you change to "Same group" and "Different groups"?
```suggestion
    :label="data.grouping ? 'Same group' : 'Different groups'"
```

Reply:
Done